plugin/src/steps/check-plugin-requirements.ts

```diff
@@ -206,8 +218,9 @@ ${firstError.debugMessage || firstError.message}
     })
     .catch((e) => {})
 
-const isWpGatsby = async () =>
+const isWpGatsby = async ({ url }: { url: string }) =>
   fetchGraphql({
+    url,
```

Comment:
same as above :)

Reply:
just saw these review comments now for some reason? they must have appeared after i was done for the day, oops! the above you refer to is gone but i know what you're referring to